server/src/test/java/org/elasticsearch/action/admin/indices/segments/IndicesSegmentResponseTests.java

```diff
@@ -68,7 +68,7 @@ public void testChunking() {
                 0,
                 Collections.emptyList()
             ),
-            response -> 11 * response.getIndices().size() + 4
+            response -> 11 * response.getIndices().size() + 5
```

Comment:
Why did the size change?

Reply:
I think this is because in the original code startObject was in the same chunk as the "body", now it is on its own chunk. @thecoop can you confirm?

Reply:
Yes - for example, https://github.com/elastic/elasticsearch/pull/113125/files#diff-fcc228eaa368ab5dfb026ddf8dc249c16ccf8450414d1616645832bf8ae0d163R38, the `startObject()` is in the same chunk as the `buildBroadcastShardsHeader`, but in the new code, the `.object()` uses a separate chunk for the `startObject()`